### PR TITLE
Add azure infra code

### PR DESCRIPTION
**Pull request by dsrawat984**
Creates azure resources given below.
1. vnet
2. subnet
3. network interface
4. master vm
5. node vm

### CODE REVIEW

File: tests/azure/main.tf [hunk 1]
```diff
@@ -0,0 +1,203 @@
+provider "azurerm" {
+  features {}
+  subscription_id = "XXXXXXXXXXXXXXXXXXXXXXXXXXX"
```
Comment (costrouc) [inline tests/azure/main.tf:3]: Could you allow the option for these to be pased in via environment variables? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference. Just want to discourage hard coding secrets in files.

**Review by costrouc — CHANGES_REQUESTED** [review]
Thanks! Could you add a README in this directory with docs on how to deploy this? Don't need in depth details but e.g.
1. set these environment variable and create id rsa key in directory or point to keys
2. install terraform version X+
3. terraform init ... terraform apply

Directions on how to access the cluser e.g. ssh.

File: tests/azure/main.tf (hunk 1 above)
Comment (dsrawat984) [reply tests/azure/main.tf:3]: Sure, will do it.

**Comment by dsrawat984** [comment]
README file needs to be generated

File: tests/azure/main.tf (hunk 1 above)
Comment (dsrawat984) [reply tests/azure/main.tf]: I have updated the README file.

**Pull request closed without merging**